assets/sass/widgets/_widget-context.scss

```diff
@@ -24,4 +24,8 @@
 
 .googlesitekit-widget-context--last {
 	min-height: 80vh;
+
+	@media screen and (min-height: 1200px) {
```

Comment:
1200 is somewhat of an arbitrary choice here, but I tested that it works at 1199. This way, only screens taller than full HD (1080px) get the fix that only they need.